dev/docker-compose-sdp-anchor.yml

```diff
@@ -194,6 +194,25 @@ services:
       - KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092,EXTERNAL://localhost:9094
       - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,EXTERNAL:PLAINTEXT,PLAINTEXT:PLAINTEXT
       - KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
+    healthcheck:
+      test: /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:9092 --list || exit -1
+      start_period: 10s
+      interval: 10s
+      timeout: 10s
+      retries: 5
+
+  kafka-init:
+    image: docker.io/bitnami/kafka:3.6
+    entrypoint: [ "/bin/bash", "-c" ]
+    depends_on:
+      kafka:
+        condition: service_healthy
+    command: |
+      "
+        /opt/bitnami/kafka/bin/kafka-topics.sh --create --if-not-exists --topic events.receiver-wallets.new_invitation --bootstrap-server kafka:9092
+        /opt/bitnami/kafka/bin/kafka-topics.sh --create --if-not-exists --topic events.payment.payment_completed --bootstrap-server kafka:9092
+        /opt/bitnami/kafka/bin/kafka-topics.sh --create --if-not-exists --topic events.payment.ready_to_pay --bootstrap-server kafka:9092
+      "
```

Comment:
Hey @CaioTeixeira95, could you plz add comments here and in [internal/events/handler.go](https://github.com/stellar/stellar-disbursement-platform-backend/blob/f8e0abd694b5b9b433d78809d4d15a1a5520ba84/internal/events/handler.go#L7-L12) reminding devs to keep these topics synchronized between the Go code and the docker-compose file?

Reply:
Could you please take a look if the comment is ok?

PTAL - [15150a9](https://github.com/stellar/stellar-disbursement-platform-backend/pull/200/commits/15150a9070ba7a29be1cb7050c0b0d568b7f341f)